app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -211,6 +215,14 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
     private Media media;
     private ArrayList<String> reasonList;
 
+    /**
+     * Height stores the height of the frame layout as soon as its intialised and updates itself on
+     * configuration change.
+     * Used to adjust aspect ratio of image when length of the image is too large.
+     */
+    private int heightFrameLayuot;
```

Comment:
Typo: Layout
Actually, I think that `frrameLayoutHeight` would be more understandable. Or am I mistaken?

Reply:
done ;)